src/settings.ts

```diff
@@ -89,6 +89,7 @@ export const DUST_STORM_TEXT_2 =
 
 // How often to refresh events schedule
 export const REFETCH_SCHEDULE_MS = 10 * 60 * 1000; // 10 mins
+export const SCHEDULE_LONG_EVENT_LENGTH_MIN = 60;
```

Comment:
I believe we should add one more const here for the events that should not be expandable. (45 min)
If the event is 45min or longer it should appear as is.

Reply:
added in 64c0c27